modules/cnv-troubleshooting-incorrect-policy-config.adoc

```diff
@@ -0,0 +1,213 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-troubleshooting-node-network.adoc
+
+[id="cnv-troubleshooting-incorrect-policy-config_{context}"]
+= Troubleshooting: Incorrect NodeNetworkConfigurationPolicy configuration
+
+Applying a NodeNetworkConfigurationPolicy to the cluster is an easy way to apply changes to the node network configuration consistently across your entire cluster.
+However, this also means it is easy to apply an incorrect configuration.
+
+This troublshooting example investigates how to identify and correct the configuration of a failed network Policy.
+
+In this example, a Linux bridge Policy is applied to an example cluster that has 3 master nodes and 3 worker nodes. The Policy fails to be applied because it references an incorrect interface. The error can be found by investigating the available nmstate resources and corrected by updating the Policy. 
+
+.Procedure
+
+. Create a Policy and apply it to cluster. The following example creates a simple bridge on the `ens01` interface:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: ens01-bridge-testfail
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        description: Linux bridge with the wrong port 
+        type: linux-bridge
+        state: up
+        ipv4:
+          dhcp: true
+          enabled: true
+        bridge:
+          options:
+            stp:
+              enabled: false
+          port:
+            - name: ens01
+----
++
+----
+nodenetworkconfigurationpolicy.nmstate.io/ens01-bridge-testfail created
+----
+
+. It can take up to 20 seconds for the policy to apply, after which we can check the status of the Policy to ensure it has been successfully configured:
```

Comment:
s/we/you
or
"Wait approximately 20 seconds for the Policy to apply to the cluster. Then, verify that the Policy was successfully configured by running the following command:"

Reply:
Dropped the time reference and updated to 'Verify the status of the Policy by running the following command:'